kong/plugins/jwt/handler.lua

```diff
@@ -226,6 +224,12 @@ local function do_authentication(conf)
 end
 
 
+local JwtHandler = {
```

Comment:
Style: I recommend we leave this on top of the file for easier to find and consistency reasons.

Reply:
+1 on this